VoucherifyClient.php

```diff
@@ -143,6 +143,25 @@ public function redeem($code, $trackingId) {
         }
 
         /**
+         * @param string|array $code Voucher code or array with voucher (code), customer profile and order amount
+         * @param string|null $trackingId Provided tracking id
+         *
+         * Validate voucher
+         *
+         * @throws Voucherify\ClientException
+         */
+        public function validate($code, $trackingId = NULL) {
+            $context = array();
+            if (is_array($code)) {
+                $context = $code;
+                $code = $context["voucher"];
+                unset($context["voucher"]);
+            }
+
+            return $this->apiRequest("POST", "/vouchers/" . urlencode($code) . "/validate", ["tracking_id" => $trackingId], $context);
```

Comment:
`$tracking_id` should be sent as `$context.customer.source_id` - please check updated documentation:
https://docs.voucherify.io/reference#validate-voucher